lib/tasks/csv_to_seed.rake

```diff
@@ -0,0 +1,50 @@
+require 'csv'
+require_relative '../../app/models/application_record'
+require_relative '../../app/models/customer'
+
+desc "Loads the csv files and seeds the database"
+
+task csv_to_seed: :environment do
+
+  CSV.foreach(Rails.root.join('lib/customers.csv'), headers: true) do |row|
+    Customer.new({
+      first_name: row[1],
+      last_name: row[2]
+    })
+    require 'pry'; binding.pry
+    
+  end
+  puts "Loaded customers.csv"
+
+  CSV.foreach(Rails.root.join('lib/merchants.csv'), headers: true) do |row|
+  
+  Merchant.create({
+    name: row[1],
+  })
```

Comment:
working on figuring out how to overwrite the `id` in the tables with the `id` provided in the csv files